app/assets/stylesheets/discovery/show.scss

```diff
@@ -0,0 +1,3 @@
+.panel-space {
+  margin-bottom: 1rem !important;
+}
```

Comment:
Can you pull this inline? Discovery is a small plugin, we won't have much UX and this actually involves some packaging work I would like to avoid.

Reply:
if you are adding more css, maybe its best to move straight to webpack based css vs asset pipeline. I would search if none of our existing patternfly predefined classes provide a similar experience

Reply:
Let's create a refactoring ticket please, I am a bit fuzzy about webpack, @ik5 are you able to create a ticket and describe what's needed? I would just copy and paste @ohadlevy 's text from here ;-)

Reply:
we can rewrite parts of discovery - I actually think the whole index page should move from a table to a list view https://www.patternfly.org/pattern-library/content-views/list-view/, and thats probably best discussed with @rohoover first.

IMHO you can just use webpack/scss/es6 etc as core does, discovery has a fairly small UI footprint and we could easily convert it.

Reply:
> What do you think?

I think let's not do it in this PR, I would like to have the small improvements finished first, because the whole refactoring can take some time. We can take the time for proper redesign.

Reply:
@lzap i didn't meant to do it on this PR :)
But the whole idea that was talked about here.